OpenRA.Mods.Common/Traits/Multipliers/ProductionCostMultiplier.cs

```diff
@@ -0,0 +1,44 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Modifies the production cost of this actor for a specific queue or when a prerequisite is granted.")]
```

Comment:
-or

Reply:
Queue is optional too, i feel like your sentence says it isn't.

Reply:
Point taken.